__tests__/app.test.js

```diff
@@ -7,6 +7,24 @@ const seed = require("./../db/seeds/seed")
 beforeEach(() => seed({ topicData, userData, commentData, articleData }))
 afterAll(() => db.end())
 
+//api
+describe("GET /api", () => {
+    test("status: 200, responds with JSON object containing all available endpoints in the API", () => {
```

Comment:
You should check the response has the endpoints, maybe require them in here too and check they match